runtime/near-vm-runner/src/runner.rs

```diff
@@ -47,6 +48,21 @@ pub fn run<'a>(
     )
 }
 
+/// Validate and run the specified contract with a specific VM implementation.
+///
+/// This is the entry point for executing a NEAR protocol contract. Before the entry point (as
+/// specified by the `method_name` argument) of the contract code is executed, the contract will be
+/// validated (see [`prepare::prepare_contract`]), instrumented (e.g. for gas accounting), and
+/// linked with the externs specified via the `ext` argument.
+///
+/// [`VMContext::input`] will be passed to the contract entrypoint as an argument.
+///
+/// The gas cost for contract preparation will be subtracted by the VM implementation.
+///
+/// # Panics
+///
+/// Calling this function with a `VMKind` that has not been enabled at the compile time will cause
+/// this function to panic.
```

Comment:
Forgot to submit comment, will expand tommorrow, but TL;DR is that we should say that `run_vm` isn't really a public API, unlike `run`.

Reply:
So, yeah, let's change this to something ``Internal variant of [`run_vm`] which allows customizing the wasm VM being used for testing purposes.``